docs/pages/blog/mui-x-sep-2024-price-update.md

```diff
@@ -2,57 +2,75 @@
 title: Upcoming changes to MUI X pricing in 2024
 description: Check out the new pricing updates and how to transition to the new model.
 date: 2024-08-01T08:00:00.000Z
-authors: ['josefreitas']
+authors: ['josefreitas', 'oliviertassinari']
 tags: ['MUI X', 'Product']
 manualCard: true
 ---
 
-We're excited to share some important updates to our product offerings and pricing structure. This change reflects the expanded feature set and helps us continue delivering high-quality components to you and your users.
+Today, we are sharing some important updates to the MUI X product offerings and pricing structure.
+These changes are made to reflect the expanded feature set and help support the delivering of high-quality components to you and your users.
 
 ## Changes to the Pro plan
 
-### Current Pricing
+### Current pricing
 
-Under our current pricing model, if you licensed the Pro plan for 50 developers, you only paid for 10 seats. This concept was introduced when we offered only the Data Grid Pro with a much smaller feature set.
+Under the current pricing model, you only purchase a license for the first 10 developer.
+So to license the Pro plan for 50 developers you only need to purchase 10 licenses.
```

Comment:
```suggestion
So to license the Pro plan for 50 developers you would purchase 10 licenses.
```

Btw, regarding banning "seats", I'm not sure if mixing "licenses" in plural with "license" in singular doesn't make it more confusing (and maybe even legally ambiguous)

Reply:
Yeah, to get a project licensed, each developer needs to be licensed. One word for two different concepts. I think we should address this a whole though.